localization/english/wc_game_concepts_l_english.yml

```diff
@@ -60,4 +60,7 @@
  game_concept_wc_incapable_desc:0 "Incapable characters cannot take actions independently due to severe illness or being isolated."
 
  game_concept_demonic_government:0 "$demonic_government_adjective$ Government"
- game_concept_demonic_government_desc:0 "A [dark_government|E] available to [GetTrait('being_demon').GetName( GetNullCharacter )] with $doctrine_disorder_magic_name$ accepted.\nHaving high [dark_authority|E], they get less [tyranny|E], slow down [dread|E] decay and their [armies|E] move faster."
\ No newline at end of file
+ game_concept_demonic_government_desc:0 "A [dark_government|E] available to [GetTrait('being_demon').GetName( GetNullCharacter )] with $doctrine_disorder_magic_name$ accepted.\nHaving high [dark_authority|E], they get less [tyranny|E], slow down [dread|E] decay and their [armies|E] move faster."
```

Comment:
I'm not sure if Demonic still gain less Tyranny for higher Dark Authority level.

Reply:
But that's not the topic of this PR, is it? This is about Old Gods.

Reply:
> But that's not the topic of this PR, is it? This is about Old Gods.

You are right, it was in the last dark government PR. I'll put ahead a separate issue for it if @zumbak04 wants it that way.

Reply:
> You are right, it was in the last dark government PR. I'll put ahead a separate issue for it if @zumbak04 wants it that way.

Nah, it's fine.

Reply:
Though, nah, I'll fix it on another branch.

Reply:
Fixed in a5474e3d6e311e1a943ebed51772f83c64aea593.